pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -1838,8 +1838,30 @@ void FullscreenUI::DrawInputBindingButton(
 	}
 	else if (hovered && (ImGui::IsItemClicked(ImGuiMouseButton_Right) || ImGui::Shortcut(ImGuiKey_NavGamepadMenu)))
 	{
-		bsi->DeleteValue(section, name);
-		SetSettingsChanged(bsi);
+		SmallString current_value = bsi->GetSmallStringValue(section, name);
+		if (!current_value.empty())
+		{
+			bsi->DeleteValue(section, name);
+			SetSettingsChanged(bsi);
+		}
```

Comment:
I believe  `bsi->DeleteValue()` can handle missing entries itself

Is this to avoid an unneeded `SetSettingsChanged()` or some other reason?